dts/arm/gigadevice/gd32f403/gd32f403.dtsi

```diff
@@ -16,8 +17,8 @@
 		#size-cells = <0>;
 
 		cpu0: cpu@0 {
-			device_type = "cpu";
```

Comment:
its ok to keep `device_type = "cpu";`

Reply:
compatibility with IEEE 1275 is not needed here, this is why I removed it.